main.go

```diff
@@ -362,12 +363,17 @@ func main() {
 		os.Exit(1)
 	}
 
-	if err := (&ec2API.ENICleaner{
-		EC2Wrapper:  ec2Wrapper,
+	cleaner := &eniCleaner.ClusterENICleaner{
 		ClusterName: clusterName,
-		Log:         ctrl.Log.WithName("eni cleaner"),
-		VPCID:       vpcID,
-	}).SetupWithManager(ctx, mgr, healthzHandler); err != nil {
+	}
+	cleaner.ENICleaner = &eniCleaner.ENICleaner{
```

Comment:
Is cluster ENICleaner for cluster wide 30 minutes cleanup? I am trying to understand the pattern here.